docs/writing-tests/test-addon.mdx

```diff
@@ -190,7 +196,39 @@ When the addon is set up automatically, it will create or adjust your Vitest con
 
 ## Usage
 
-There are three primary ways to run tests using the Vitest plugin:
+There are multiple ways to run tests using the addon:
+
+### Storybook UI
+
+The easiest way to run tests is through the Storybook UI. To run all tests for your components, press the Run tests button in the test module at the bottom of the sidebar.
+
+<Video src="../_assets/writing-tests/addon-test-run-all-tests.mp4" />
+
+Alternatively, you can expand the test module to run specific types of tests individually. For those test types which have a watch mode (which will automatically re-run relevant tests upon code changes), you can toggle that on or off.
+
+![Screenshot of test module, expanded, showing test types and watch mode toggle](../_assets/writing-tests/addon-test-module-expanded.png)
+
+<Callout variant="info" icon="ℹ️">
+
+If you have the [Visual tests addon](./visual-testing.mdx) installed, you'll see an option to run Visual tests alongside [Component tests](./component-testing.mdx).
+
+![Screenshot of test module, expanded, showing Visual tests](../_assets/writing-tests/addon-test-module-expanded-with-vta.png)
+
+Other addons may also provide test types that can be run from the test module and affect the status indicators on stories and components.
+
```

Comment:
Clarify this sentence to provide information about which addons have an impact on the testing module and affect the status indicators, Or remove it altogether to avoid confusion as users don't have the same context that you have and the documentation itself does not provide any additional context.

Reply:
Fair enough. We can put it back with an example when the a11y support is added.